README.md

```diff
@@ -31,32 +31,33 @@ Each version of the plugin is published with a [specific set of versions of the
+| `addExtension`                | Whether to install the Datadog Lambda Extension as a layer. Defaults to `false`. When enabled, you must set either the environment variable `DATADOG_API_KEY`, or the configuration values `apiKMSKey` or `apiKeySecretArn`. Learn more about the [Datadog Lambda Extension Layer][8] in the documentation. **Note**: AWS only supports Lambda Extensions for [certain runtimes][9].                                                |
+| `logLevel`                    | The log level, set to `DEBUG` for extended logging.                                                                                                                                                                                                                                                                                                                                                                                 |
+| `enableXrayTracing`           | Set `true` to enable X-Ray tracing on the Lambda functions and API Gateway integrations. Defaults to `false`.                                                                                                                                                                                                                                                                                                                       |
+| `enableDDTracing`             | Enable Datadog tracing on the Lambda function. Note: This applies only to integrations using the Datadog Extension. Defaults to `true`.                                                                                                                                                                                                                                                                                             |
+| `enableDDLogs`                | Enable Datadog log collection for the Lambda function. Note: This setting has no effect on logs sent by the Datadog Forwarder. Defaults to `true`.                                                                                                                                                                                                                                                                                  |
+| `enableSourceCodeIntegration` | Enable Datadog Source Code integration for the function. Only works for Java and Go.                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
1. Source Code -> source code
2. Can you elaborate on what this actually means/does?
3. For "only works for Java and Go", I would rephrase this slightly to be a bit softer, maybe "This feature is currently only supported for functions written in Java or Go."

Reply:
Good idea, I actually originally had this but removed it because I felt it was a bit much and couldn't figure out a way to word it. Will do!